### PR TITLE
add ability to confirm password and check if passwords match

### PR DESCRIPTION
**Pull request by Gachenge**

### CODE REVIEW

File: openapi/auth/routes.py [hunk 1]
```diff
@@ -15,6 +15,9 @@ def register():
             name = data.get("name")
             email = data.get("email")
             password = data.get("password")
+            confirm_password = data.get("confirm_password")
+            if password != confirm_password:
+                return jsonify({"Error": "Password and confirm_password do not match"})
```
Comment (Godhanded) [inline openapi/auth/routes.py]: This would return an error but still give 200 status ode,Include status code to response

**Review by Godhanded — CHANGES_REQUESTED** [review]
Fix status code errors

File: openapi/auth/routes.py (hunk 1 above)
Comment (sshdopey) [reply openapi/auth/routes.py]: Guardian of the codebase

Comment (Gachenge) [reply openapi/auth/routes.py]: I have added the status code

**Review by Godhanded — APPROVED** [review]
approved

**Pull request merged**